dev/tasks/tasks.yml

```
@@ -1040,7 +1040,7 @@ tasks:
         TEST_INTEGRATION_JAVA: 0
         PYTEST_ADDOPTS: "-k 'not test_cancellation'"
       target: {{ target }}
-      github_runner: "self-hosted"
+      runs-on: ["self-hosted", "macOS", "arm64"]
```

You renamed the param here, I guess that did not error because the template has a default.
```suggestion
      github_runner: '["self-hosted", "macOS", "arm64"]'
```